ocs_ci/ocs/ocp.py

```diff
@@ -511,6 +511,10 @@ def wait_for_resource(
                         resource_name, column, retry=retry, wait=sleep,
                     )
                     if status == condition:
+                        log.info(
+                            f"status of {resource_name} at {column}"
+                            f" reached condition!"
```

Comment:
You can omit `f` here as you don't expand any war in this line

Reply:
Done